pkgs/by-name/la/labwc-menu-generator/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, glib
+, gitUpdater
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "labwc-menu-generator";
+  version = "unstable-2023-10-31";
+
+  src = fetchFromGitHub {
+    owner = "labwc";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "labwc-menu-generator";
```

https://github.com/NixOS/nixpkgs/issues/277994

---

Done.